odbc/src/api/data.rs

```diff
@@ -385,12 +385,12 @@ impl IntoCData for Bson {
                     (
                         NaiveDate::parse_from_str(s, "%F")
                             .map_err(|_| ODBCError::InvalidDatetimeFormat)?,
-                        NaiveTime::from_hms(0, 0, 0),
+                        NaiveTime::from_hms_opt(0, 0, 0).ok_or(ODBCError::InvalidTime)?,
```

Comment:
I'm curious in these cases if we need the `InvalidTime` error, or if we should just `unwrap`. Looking at the docs, it seems this _should_ be guaranteed to be some: 
```assert!(from_hms_opt(0, 0, 0).is_some());```

Reply:
I was debating whether to unwrap or return an error.  I think since we're hard-coding the values it will be okay to unwrap, I'll make the change.